src/test/common/steps/pages.js

```diff
@@ -61,6 +63,7 @@ class Pages {
     this.cardAddress = new CardAddress(this.driver)
     this.check = new Check(this.driver)
     this.confirm = new Confirm(this.driver)
+    this.confirmUpdated = new ConfirmUpdated(this.driver)
```

Comment:
You might want to add it to the `allPages` map too in case it diverges from the Confirm page at any point.

Reply:
Done.